.gitignore

```diff
@@ -1,5 +1,3 @@
-firebaseCredentials.json
```

Comment:
Why did you remove this from gitignore?

Reply:
It was already in there at the bottom of .gitignore from when I pulled! Just didn't see it

Reply:
Okay sounds good!